src/test/java/com/fullteaching/e2e/no_elastest/common/BaseLoggedTest.java

```diff
@@ -347,7 +368,7 @@ protected void waitSeconds(int seconds) {
         try {
             Thread.sleep(1000L * seconds);
         } catch (InterruptedException e) {
-            e.printStackTrace();
+            log.error("Thread.sleep interrupted");
```

Comment:
It is not frequent to have exceptions when doing a sleep, but, why this does not fails the test?